psi4/driver/driver.py

```diff
@@ -1802,7 +1802,57 @@ def gdma(wfn, datafile=""):
             f.write("Limit %d\n" % core.get_option('GDMA', 'GDMA_LIMIT'))
             f.write("Start\n")
             f.write("Finish\n")
-    core.run_gdma(wfn, commands)
+
+    # from outside the Psi4 ecosystem
+    from qcelemental.util import which_import
+    if not which_import("gdma", return_bool=True):
+        raise ModuleNotFoundError('Python module gdma not found. Solve by installing it: `conda install -c conda-forge gdma` or recompile with `-DENABLE_gdma`')
+    import gdma
+
+    min_version = "2.3.3"
+    from pkg_resources import parse_version
+    if parse_version(gdma.__version__) < parse_version(min_version):
+        raise ModuleNotFoundError(f"GDMA version {min_version} is required at least. Version {gdma.__version__} was found.")
+
+    core.prepare_options_for_module("GDMA")
+
+    gof = core.get_output_file()
+    gdma.run_gdma(gof, commands)
+    core.set_output_file(gof, True)
+    core.reopen_outfile()
+
+    nsites = gdma.get_nsites()
+    maxorder = 0
+    for site in range(1, nsites + 1):
+        site_order = gdma.get_order(site)
+        maxorder = max(maxorder, site_order)
```

Comment:
```suggestion
    maxorder = max(gdma.get_order(site) for site in range(1, nsites + 1))
```

Reply:
good iea -- I was translating directly from the c++